azure-pipelines.yml

```diff
@@ -47,6 +52,22 @@ jobs:
       scikit-learn
       scipy
     displayName: 'Install dependencies'
+  # TODO: recent rdkit is not on PyPI
+  - script: >-
+      python -m pip install
```

Comment:
When H5MD in PR #2787 is added, we will also need to add h5py, but that's already working in PR #2787 – apparently, hdf5 libraries are available.

Reply:
Ah, that's good news, if we're down to just 1 `conda`-only binary that would be good.